app/src/main/java/com/example/wordsapp/LetterListFragment.kt

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.wordsapp
+
+import android.os.Bundle
+import android.view.*
```

Comment:
add necessary imports and remove *

Reply:
This appears to be due to default behavior in Android Studio 4.1.1. When referencing types in `android.view`, they're imported individually. As this file uses five `android.view.SomeType`s (LayoutInflater, Menu, ViewGroup, View, MenuItem) the import is changed to `android.view.*` when the fifth one (MenuItem) is used in code, but will otherwise import them individually if there are four or fewer.

Is this the expected behavior? We can possibly change to import all 5 individually, but given the default import settings, when the student creates this file, the import will likely be shortened to `android.view.*` on their end, thus not matching the solution code. Thoughts?

Reply:
It is true also the compiler will optimize this. However, codelabs have their own style which doesn't necessarily match with AS defaults. You can change the behavior from the link I pasted below.
Please also submit a PR to fix the solution code.

https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports